brave/brave-core
# feat(wallet) Refactor NFT IPFS state to use RTK queries and mutations

Resolves https://github.com/brave/brave-browser/issues/32753
https://github.com/brave/brave-browser/issues/32570



## Submitter Checklist:

  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run lint`, `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:
### Test 1
- Add a bunch of NFTs from different networks
- Ensure you have at least one NFT that can be pinned via IPFS
- Once NFT is pinned ensure successful NFT pinning message is shown
- Open Panel and go to NFT tab
- The banner should have the same state as wallet page

### Test 2:

- Add a bunch of NFTs
- Hide one of the NFT from Wallet page
- Open Panel and go to NFT tab
- The Hidden NFT should appear in the Hidden tab
